MdePkg/Include/Pi/PiMultiPhase.h

```diff
@@ -20,6 +20,13 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 #include <Pi/PiStatusCode.h>
 #include <Pi/PiS3BootScript.h>
 
+//
+// PI Specification Version Information
+//
+#define PI_SPECIFICATION_MAJOR_REVISION  1
+#define PI_SPECIFICATION_MINOR_REVISION  80

```

Comment:
This changes the PEI, DXE, and MM table revisions to 1.80.  Have all spec changes up to 1.80 been verified to be implemented in the edk2 repo?  Please provide details on the list of features/changes that have been verified.

Reply:
@mdkinney 

**These are the items in PI 1.80 specification and their related commit in the edk2 repo:**

- 2103 Adding EFI_DELAYED_DISPATCH_FUNCTION definition and explanation - https://github.com/tianocore/edk2/commit/f3b0ee0cee8d0c4356af64cb2d544d7e56d861a3

- 2119 HOB Update for Memory Ranges Capable of Being Protected by CPU Cryptographic Capabilities - https://github.com/tianocore/edk2/commit/b538d6a1b1d79d0dc2fd68da9e19b5e257d3dc5a

- 2147 EFI_RESOURCE_ATTRIBUTE_SP not defined in Resource Descriptor HOB - https://github.com/tianocore/edk2/commit/b538d6a1b1d79d0dc2fd68da9e19b5e257d3dc5a

- 2160 MP Services 2 PPI Section - https://github.com/tianocore/edk2/commit/a430589cb5c79b706e60b0d4da9adfc120d5c65b

- 2161 Introduce unaccepted memory type - https://github.com/tianocore/edk2/commit/6fd2d58d5d59591daec738340e95dc330193afd8

- 2340 Introduction of EFI_SW_EC_FRAGMENTED_MEMORY_MAP Status Code - https://github.com/tianocore/edk2/commit/e7667b7b9ab8b1abc505c4a00dbcbcc22818fea3

- 2341 Introduction of EFI_PERIPHERAL_TPM Peripheral Subclass Definition - https://github.com/tianocore/edk2/commit/7c703f0a8a21216a4065953d783a9f6ef28747e9

- 2351 update PI spec revision Section - This PR

Reply:
I see some tables being updates from 1.6 to 1.8.  Is this the full list of changes from 1.6 to 1.8?

Reply:
@mdkinney The above comment lists the changes from 1.7B to 1.8. The rest are:  
  
**1.7:**

- 1848 PEI Core PEIM Migration Support Change - https://github.com/tianocore/edk2/commit/9bedaec05b7b8ba9aee248361bb61a85a26726cb
- 1856 SecCore/PeiCore BFV Requirement Change - https://github.com/tianocore/edk2/commit/9b23c7ba9c599cd8208518d41d6a1d51713c0f9d 
- 1860 MM MP Protocol issues - https://github.com/tianocore/edk2/commit/51dd408ae1022e5c9378492451a62b38d5b101c5
- 1885 Add extended data for EFI_SW_DXE_BS_EC_BOOT_OPTION_LOAD_ERROR - https://github.com/tianocore/edk2/commit/2b005f41fdf1a5783eed0d0304c8d9c114acea68
- 1889 New Status Codes - https://github.com/tianocore/edk2/commit/f8683a4ba057c840874ebeb09043916288573290 	
- 1891 PEI delayed dispatch - https://github.com/tianocore/edk2/commit/4ac245767341d9203b5b35b26d091a7f5e8534f9
- 1892 new pi spec revision - https://github.com/tianocore/edk2/commit/93ddc0d133ee793101a2bc3ce587f275b3f11915
- 1955 set (*Attributes) when EFI_PEI_GET_VARIABLE2 returns EFI_BUFFER_TOO_SMALL and Attributes is non-NULL - https://github.com/tianocore/edk2/commit/91a33d41139b4ab8c446567cbe09e10501224277
- 1967 New architectural PPI for PI PEI Core FV Location - https://github.com/tianocore/edk2/commit/c93c0dc1d56fe78358cddd63c988ba316cf07cde
- 1969 Incompatible Pci Ignore Option Rom - https://github.com/tianocore/edk2/commit/c477b2783f6d95b62266f4fff9b0c286ac1d7bb8

**1.7A:**

- 1663 SmmSxDispatch2 to Register() is not clear - Documentation Update
- 1736 Specification of EFI_BOOT_SCRIPT_WIDTH in Save State Write - Documentation Update
- 1993 Allow MM CommBuffer to be passed as a VA - https://github.com/tianocore/edk2/commit/9ad2b981bd6469b163ec0c177c9d549bfe22147f
- 2017 EFI_RUNTIME_EVENT_ENTRY.Event should have type EFI_EVENT, not (EFI_EVENT*) - https://github.com/tianocore/edk2/commit/d342d318e9da83031841b602012699039c2e99bb
- 2039 PI Configuration Tables Errata - Documentation Update
- 2040 EFI_SECTION_FREEFORM_SUBTYPE_GUID Errata - Documentation Update
- 2060 Add missing EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL_PCI_ADDRESS definition - Already part of https://github.com/tianocore/edk2/tree/master/MdeModulePkg/Bus/Pci/PciHostBridgeDxe
- 2063 Add Index to end of PI Spec - Documentation Update
- 2071 Extended cpu topology - https://github.com/tianocore/edk2/commit/15ac284815b1e1d5fe8dbc737129f82c9a92649b

**1.7B:**

- 2101 Fix DXE_SPECIFICATION_MINOR_REVISION and PEI_SERVICES_REVISION definitions - This PR
- 2110 Status Code Class definition, incorrect comment on OEM range - Documentation Update
- 2213 Clarify PEI Memory Allocation Rules - Documentation Update
- 2364 Inclusive language implementation - added new disclaimer section - Documentation Update

Reply:
@mdkinney Please let me know if there are any further concerns with this PR.

Reply:
Thank you for the updated list.

For the items listed as "Documentation Update", is there any impact to the function headers that requires a PR?

Reply:
Hi @mdkinney 

There were two items which needed comment changes. That has been done and committed as part of this PR.

Reply:
Hi @mdkinney 

Could you let me know if there are any further concerns on this PR